SwedbankPaySDK/Classes/SwedbankPaySDK+Extensions/MerchantBackendConfiguration.swift

```diff
@@ -0,0 +1,207 @@
+            api = MerchantBackendApi(
+                session: session,
+                domainWhitelist: MerchantBackendConfiguration.makeDomainWhitelist(
+                    backendUrl: backendUrl, domainWhitelist: domainWhitelist
+                ),
+                requestDecorator: headers.map(SimpleRequestDecorator.init(headers:))
+            )
+            rootLink = RootLink(href: backendUrl)
+            
+            self.callbackScheme = callbackScheme ?? MerchantBackendConfiguration.getDefaultCallbackScheme()
+            self.additionalAllowedWebViewRedirects = additionalAllowedWebViewRedirects
+        }
+        
+        private static func makeSession(pinPublicKeys: [PinPublicKeys]?) -> Session {
+            if let pinPublicKeys = pinPublicKeys, !pinPublicKeys.isEmpty {
+                var pinEvaluators: [String: PublicKeysTrustEvaluator] = [:]
+                for certificate in pinPublicKeys {
+                    pinEvaluators[certificate.pattern] = PublicKeysTrustEvaluator(
+                        keys: certificate.publicKeys,
+                        performDefaultValidation: true,
+                        validateHost: true
+                    )
+                }
+                return Session(
+                    configuration: URLSessionConfiguration.default,
+                    serverTrustManager: ServerTrustManager(
+                        evaluators: pinEvaluators
+                    )
+                )
+            } else {
+                return Session()
+            }
+        }
+        
+        private static func makeDomainWhitelist(
+            backendUrl: URL,
+            domainWhitelist: [WhitelistedDomain]?
+        ) -> [WhitelistedDomain] {
+            if let domainWhitelist = domainWhitelist, !domainWhitelist.isEmpty {
+                return domainWhitelist
+            } else {
+                let domain = WhitelistedDomain.init(
+                    domain: backendUrl.host,
+                    includeSubdomains: true
+                )
+                return [domain]
+            }
+        }
+        
+        private static func getDefaultCallbackScheme() -> String {
+            let urlTypes = Bundle.main.object(forInfoDictionaryKey: "CFBundleURLTypes") as? [Any]
+            let urlTypeDicts = urlTypes?.lazy.compactMap { $0 as? [AnyHashable : Any] }
+            guard let callbackUrlType = urlTypeDicts?.filter({
+                $0[callbackURLTypeKey] as? Bool == true
+            }).first else {
+                fatalError("Unable to infer callback scheme: No URL type marked as Swedbank Pay callback. Please add a URL type with a unique name, a single unique scheme, and an additional property with the key \(callbackURLTypeKey), type Boolean, and value YES")
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 264 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)